packages/cli/src/commands/hydrogen/generate/route.ts

```diff
@@ -72,6 +73,10 @@ export default class GenerateRoute extends Command {
 
     const {route} = args;
 
+    const remixConfig = await getRemixConfig(directory);
+    const isV2RouteConvention = !!remixConfig.future?.v2_routeConvention;
```

Comment:
Hmm, will this still be there when v2 is released, or how can we determine this when v2 releases?

Reply:
Now checking if Remix is v2 as well 👍